Update Algemene Plaateslijke Verordening Tedeapolis.md

PapaXilion opened the pull request:
Aanpassing Art. 18. Dit was eerder Alt Accounts, maar dit is niet de juiste benaming. Alt Characters klopt beter. Daarnaast de punten iets uitgebreid om Alt Accounts wèl op te nemen als puntje.

PapaXilion committed:
Update Algemene Plaateslijke Verordening Tedeapolis.md

Aanpassing Art. 18. Dit was eerder Alt Accounts, maar dit is niet de juiste benaming. Alt Characters klopt beter. Daarnaast de punten iets uitgebreid om Alt Accounts wèl op te nemen als puntje.

Willem-Dekker commented on docs/Algemene Plaateslijke Verordening Tedeapolis.md:
```diff
@@ -172,11 +172,13 @@ De overtreding beschreven in lid 1 staat bekend als “spam”
+2. Hiernaast is het ook verboden om meer dan 3 alts aan te maken. Het aanmaken van meer dan 3 alts zal resulteren in het verwijderen van alle 3 de characters. Wanneer er geld overgezet is zal hiernaast ook de straf zoals beschreven in lid 4 gegeven worden.
+3. Het is toegestaan om een alt character te maken om, bijvoorbeeld, criminele activiteiten te ondernemen. Wanneer een van je characters bij de Overheid werkt, is het *niet* toegestaan om bij een gang te gaan.
+4. Indien de regel zoals beschreven in lid 1 of 3 overtreden wordt zal dit bestraft worden met een straf volgens de 7e categorie, hiernaast zullen alle accounts volledig verwijderd worden.
+5. Hoewel Alt Characters toegestaan zijn, is het verboden om alt accounts te maken, of te multiboxen met alt characters. Dit zal worden bestraft met een ban van alle (alt) accounts.
```
wat bedoel je hier mee.

PapaXilion replied:
Deze stond al in de APV, ik heb 'm echter omgegooid naar alt *characters* in plaats van Accounts.

PapaXilion replied:
Oh sorry, verduidelijking: alt accounts zijn verboden (dus meerdere accounts aanmaken om op te spelen). Multiboxen is het tegelijk draaien van die accounts op 1 pc.

Willem-Dekker commented on docs/Algemene Plaateslijke Verordening Tedeapolis.md:
```diff
@@ -172,11 +172,13 @@ De overtreding beschreven in lid 1 staat bekend als “spam”
+1. Het is verboden alt characters aan te maken met de intentie het start bedrag over te schrijven naar het account waar normaal op gespeeld wordt.
+2. Hiernaast is het ook verboden om meer dan 3 alts aan te maken. Het aanmaken van meer dan 3 alts zal resulteren in het verwijderen van alle 3 de characters. Wanneer er geld overgezet is zal hiernaast ook de straf zoals beschreven in lid 4 gegeven worden.
+3. Het is toegestaan om een alt character te maken om, bijvoorbeeld, criminele activiteiten te ondernemen. Wanneer een van je characters bij de Overheid werkt, is het *niet* toegestaan om bij een gang te gaan.
+4. Indien de regel zoals beschreven in lid 1 of 3 overtreden wordt zal dit bestraft worden met een straf volgens de 7e categorie, hiernaast zullen alle accounts volledig verwijderd worden.
```
voor wipes is Cat 8 toch ook?

PapaXilion replied:
Heb je gelijk in! Mag dan als Cat 8 :)

PapaXilion committed:
Update Algemene Plaateslijke Verordening Tedeapolis.md

PapaXilion committed:
Update Algemene Plaateslijke Verordening Tedeapolis.md

Aantal tekstuele aanpassingen.
Daarnaast voorstel om Art. 38 over te hevelen naar het Wetboek. Gezichtsbedekkingsverboden zijn een roleplay aangelegenheid.

PapaXilion committed:
Update Algemene Plaateslijke Verordening Tedeapolis.md
